### ISSUE TITLE
Update to latest PostCSS required

### ISSUE THREAD

**Issue by devinus** [axa-ch/metalsmith-postcss#3]
```
Your current PostCSS version is 4.1.16, but autoprefixer uses 5.0.4. Perhaps this is the source of the error below.
/Users/devin/Projects/rebelmail-website/build.js:47
    if (err) throw err;
                   ^
TypeError: css.walkAtRules is not a function
    at Processor.remove (/Users/devin/Projects/rebelmail-website/node_modules/autoprefixer/lib/processor.js:101:11)
    at plugin (/Users/devin/Projects/rebelmail-website/node_modules/autoprefixer/lib/autoprefixer.js:49:28)
    at LazyResult.run (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/node_modules/postcss/lib/lazy-result.js:197:24)
    at LazyResult.sync (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/node_modules/postcss/lib/lazy-result.js:183:32)
    at LazyResult.stringify (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/node_modules/postcss/lib/lazy-result.js:214:14)
    at LazyResult.get (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/node_modules/postcss/lib/lazy-result.js:236:25)
    at /Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/index.js:18:91
    at Array.forEach (native)
    at Ware.<anonymous> (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-postcss/index.js:17:12)
    at Ware.<anonymous> (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith/node_modules/ware/node_modules/wrap-fn/index.js:45:19)
    at next (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith/node_modules/ware/lib/index.js:85:20)
    at /Users/devin/Projects/rebelmail-website/node_modules/metalsmith/node_modules/ware/node_modules/wrap-fn/index.js:121:18
    at /Users/devin/Projects/rebelmail-website/node_modules/metalsmith-sass/node_modules/async/lib/async.js:52:16
    at done (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-sass/node_modules/async/lib/async.js:248:21)
    at /Users/devin/Projects/rebelmail-website/node_modules/metalsmith-sass/node_modules/async/lib/async.js:44:16
    at Object.callback (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-sass/lib/index.js:100:13)
    at options.success (/Users/devin/Projects/rebelmail-website/node_modules/metalsmith-sass/node_modules/node-sass/lib/index.js:293:32)
```

**Comment by sventschui** [comment]
Can you please retry using version 3.0.0?

**Comment by sventschui** [comment]
Closing as this should be fixed in 3.0.0
